zookeeper/include/zk_nn_client.h

```diff
@@ -270,6 +270,22 @@ class ZkNnClient : public ZkClientCommon {
    */
   u_int64_t get_index_within_block_group(u_int64_t storage_block_id);
 
+  /**
+   * true if the highest bit is set to 1. false otherwise.
+   * @param block_id the id of a block (or a block group)
+   * @return true or false.
+   */
+  bool is_ec_block(u_int64_t block_id);
```

Comment:
should be uint64_t, this should be changed in several places from the last PR too.

Reply:
uint64_t and u_int64_t. What are their differences? they both seem to be type aliases for the unsigned long long type.

Reply:
So I actually thought that a u_int64_t was just a typo (I had it open in sublime and it didn't recognize it as a type alias). After looking it up, u_int64_t works as well, but I think uint64_t is a bit more standard. It's perfectly fine as-is though, my bad.

Reply:
no problem